te3/features/dax-query.md

```diff
@@ -0,0 +1,78 @@
+﻿---
+uid: dax-query
+title: DAX Queries
+author: Morten Lønskov
+updated: 2023-11-09
+applies_to:
+  editions:
+    - edition: Desktop
+    - edition: Business
+    - edition: Enterprise
+---
+# DAX Queries
+
+Tabular Editor has a built in DAX query window to write and execute DAX queries against the semantic model.  
+
+A very common use case for DAX queries is the DAX query produced by the [Power BI Performance Analyzer](https://www.sqlbi.com/articles/introducing-the-power-bi-performance-analyzer/), where it is possible to copy the query of each visual for troubleshooting, debugging or detailed performance analysis. 
+
+The window can be opened while connected to a sematic model by using either the File > New > DAX Query folder or the toolbar shortcut.
+
+![Dax Query New](~/images/features/dax_query_window/create_new_dax_query.png)
+
+The built in context aware intellisense ensure that only the two valid DAX keywords are available when starting a new query: DEFINE or EVALUATE (Press Ctrl+Space to verify for yourself)
+
+## DAX Query Toolbar
+
+The DAX query window has four different toolbar options.
+
+![Dax Query New](~/images/features/dax_query_window/dax_query_toolbar.png)
+
+
+1. Execute (F5)
+2. Execute Selection (Shift+F5)
+3. Stop 
+4. Auto Execute Query
+
+The Auto Execute Query allows for keeping track of the connected semantic model and update the query results whenever something changes in the model. This can be useful for understanding e.g. how the result of a measure changes if modified. 
+
+## DAX Query Example
+
+A DAX query always return a table of results, and the simplest form of DAX query to create is one that evaluates a table within the model.
+
+```DAX
+EVALUATE
+Products
+```
+![Dax Query New](~/images/features/dax_query_window/evaluate_table.png)
+
+It is also possible to return the value of a measure but a table constructor {} is required around it measure name 
```

Comment:
"it measure name" -> "the measure name, to turn the scalar value into a 1x1 table"